content/applications/sales/point_of_sale/pricing/digital_labels.rst

```diff
@@ -0,0 +1,151 @@
+   Odoo uses digital labels from `Pricer <https://www.pricer.com/>`_.
+
+Configuration
+=============
+
+Pricer setup
+------------
+
+#. `Get in touch with Pricer <https://www.pricer.com/contact>`_ to create and configure your Pricer
+   account.
+#. Create your stores: one pricer store equates to one physical store.
+#. Create the following variables to allow sharing of information between Odoo and Pricer:
+
+   - `itemId` : the internal ID of your products
+   - `itemName` : the name of your products
+   - `price` : the price of your products, taxes included
+   - `presentation`: the template name used for Pricer tags display
+   - `currency` : the currency of your company
+   - `barcode` : the barcode associated with your products
+
+   .. warning::
+      The names for these variables must be **identical** in your Pricer database.
+
+#. Create a template used to display information on your digital tags.
+
+   .. warning::
+      Your template must be named `NORMAL`.
+
+Once your account, stores, variables, and template are configured on Pricer, you can proceed to
+the Odoo setup.
+
+.. warning::
+   The account associated with your Pricer store must have access to send API requests to Pricer.
+
+Odoo setup
+----------
+
+As a pre-requisite, :ref:`activate <general/install>` the :guilabel:`POS Pricer` module *(technical
+name: pos_pricer)* to have all the required features to use Pricer electronic tags.
+
+.. image:: digital_labels/pricer-module.png
+   :alt: Installing POS Pricer module from Apps
+
+Once the module is activated, configure your :ref:`pricer stores <pricer_tags/stores>` and associate
+:ref:`Pricer tags <pricer_tags/tags>` with your products.
+
+.. _pricer_tags/stores:
+
+Pricer stores
+~~~~~~~~~~~~~
+
+Similarly to the configuration in Pricer, you need to create one pricer store per physical location.
+To do so, go to :menuselection:`Point of Sale --> Configuration --> Pricer Stores`, click
+:guilabel:`New`, and fill in the line with the required information:
+
+- :guilabel:`Store Name`: you can put any name of your liking.
+- :guilabel:`Pricer Tenant Name`: the name of your company account in Pricer, usually followed by
+  `-country_code`. This value is provided by Pricer.
+- :guilabel:`Pricer Login` : the login of your Pricer account.
+- :guilabel:`Pricer Password` : the password of your Pricer account.
+- :guilabel:`Pricer Store ID`: the ID of the related Pricer store as defined on your Pricer
+  database.
+
+.. image:: digital_labels/pricer-stores-setup.png
+   :alt: Configuring a Pricer Store
+
+.. note::
+   - The :guilabel:`Pricer Tags` column is updated automatically when a label is linked to a
+     product.
+   - The :guilabel:`Last Update` and :guilabel:`Last Update Status` columns are updated
+     automatically when the tags are updated.
+
+.. _pricer_tags/tags:
+
+Pricer tags
+~~~~~~~~~~~
+
+For a label to display one specific product information, they both need to be associated with each
+other. To do so:
```

Comment:
Maybe it's me, but I needed to read this sentence three times to understand. It might be better to really spell it out to people, eg.:

For a label to display specific product information, the label needs to be associated with the product.

Up to you :)